src/processes/parkingspaces/internal/tests/create-offer.test.ts

```diff
@@ -22,6 +26,10 @@ describe('createOfferForInternalParkingSpace', () => {
       processStatus: ProcessStatus.failed,
       error: 'no-listing',
       httpStatus: 500,
+      response: {
+        errorCode: 'no-listing',
```

Comment:
Now that we have these error codes in onecore-types I think we should use them here as well to make it consistent :)